lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -262,7 +261,11 @@ export default function getRecipeInterface(
                         res,
                         "access",
                         response.accessToken.token,
-                        response.accessToken.expiry,
+                        // We set the expiration to 10 years, because we can't really access the expiration of the refresh token everywhere we are setting it.
+                        // This should be safe to do, since this is only the validity of the cookie (set here or on the frontend) but we check the expiration of the JWT anyway.
+                        // Even if the token is expired the presence of the token indicates that the user could have a valid refresh
+                        // Setting them to infinity would require special case handling on the frontend and just adding 10 years seems enough.
+                        Date.now() + 315360000000,
```

Comment:
We should set this to the same as the refresh token expiry time instead? Just more clean